src/EnergyPlus/WaterCoils.cc

```diff
@@ -1992,41 +1945,150 @@ namespace WaterCoils {
                 WaterConvectTerm = 0.0;
             }
             if ((AirConvectTerm > 0.0) && (WaterConvectTerm > 0.0)) {
-                WaterCoil(CoilNum).UACoilVariable = 1.0 / ((1.0 / WaterConvectTerm) + WaterCoil(CoilNum).FoulingFactor + (1.0 / AirConvectTerm));
+                WaterCoil(CoilNum).UACoilVariable = 1.0 / ((1.0 / WaterConvectTerm) + (1.0 / AirConvectTerm));
             } else {
                 // use nominal UA since variable UA cannot be calculated
-                WaterCoil(CoilNum).UACoilVariable = 1.0 / (1.0 / WaterCoil(CoilNum).UACoil + WaterCoil(CoilNum).FoulingFactor);
+                WaterCoil(CoilNum).UACoilVariable = WaterCoil(CoilNum).UACoil;
+            }
+
+            // calculate the Faulty Coil Fouling (thermal insulance) Factor using fault information
+            if (WaterCoil(CoilNum).FaultyCoilFoulingFlag &&
+                // The fault shouldn't apply during sizing.
+                (!DataGlobals::WarmupFlag) && (!DataGlobals::DoingSizing) && (!DataGlobals::KickOffSimulation) &&
+                // This was preexisting
+                !(MyUAAndFlowCalcFlag(CoilNum)))
+            {
+                // Store original value
+                WaterCoil(CoilNum).OriginalUACoilVariable = WaterCoil(CoilNum).UACoilVariable;
+
+                int FaultIndex = WaterCoil(CoilNum).FaultyCoilFoulingIndex;
+                FaultsManager::FaultPropertiesFoulingCoil &fouling = FaultsManager::FouledCoils(FaultIndex);
+                Real64 FaultFrac = fouling.FaultFraction();
+
+                if (fouling.FoulingInputMethod == FaultsManager::iFouledCoil_UARated) {
+                    // 1/UA' = Frac * (1/UAFouled) + (1-Frac) / UA
+                    WaterCoil(CoilNum).UACoilVariable = 1 / (FaultFrac / (fouling.UAFouled) + (1 - FaultFrac) / WaterCoil(CoilNum).UACoilVariable);
+                } else {
+                    // R' = R + Rfoul
+                    // Rfoul = r_air/A_air + r_water/A_water (FoulingFactor = thermal insulance [K/W, A] = Area [m2], r=fouling factor [m2.K/W]
+                    Real64 FoulingFactor = FaultFrac * (fouling.Rfw / (fouling.Aratio * fouling.Aout) + fouling.Rfa / fouling.Aout);
+                    WaterCoil(CoilNum).UACoilVariable = 1.0 / ( (1.0 / WaterCoil(CoilNum).UACoilVariable) + FoulingFactor);
+                }
+
+                // Do not allow improving coil performance
+                WaterCoil(CoilNum).UACoilVariable = min(WaterCoil(CoilNum).UACoilVariable, WaterCoil(CoilNum).OriginalUACoilVariable);
+
+                // Only for reporting purposes
+                WaterCoil(CoilNum).FaultyCoilFoulingFactor = (1.0 / WaterCoil(CoilNum).UACoilVariable) -
+                                                             (1.0 / WaterCoil(CoilNum).OriginalUACoilVariable);
+            } else {
+                WaterCoil(CoilNum).FaultyCoilFoulingFactor = 0;
```

Comment:
This is the actual calculations moved down here.  If another commit is necessary here, we should remove that commented function above.